aisdc/attacks/attribute_attack.py

```diff
@@ -98,21 +107,29 @@ def make_report(self, json_attack_formatter=None) -> dict:
             Dictionary containing all attack output.
         """
         output = {}
-        logger.info("Starting report, report_name = %s", self.report_name)
+        report_dest = os.path.join(self.output_dir, self.report_name)
+        logger.info(
+            "Starting reports, pdf report name = %s, joson report name =%s",
```

Comment:
Typo `joson` should be `json` also missing space after the equals

Reply:
Sorry, I have fixed this now.